sharktank_models/README.md

```diff
@@ -29,15 +29,9 @@ built as part of the [shark-ai project](https://github.com/nod-ai/shark-ai).
 
     python -m venv .venv
     source .venv/bin/activate
-    python -m pip install -r requirements.txt
+    python -m pip install -e .
     ```
 
-    * To use IREE from nightly pre-release Python packages:
-
-        ```bash
-        python -m pip install -r requirements-iree.txt
-        ```
```

Comment:
I think we should keep this and the "use a custom version" option on equal footing.

That will involve keeping the `requirements-iree.txt` file around and pulling these deps out of the new `setup.py`:

```diff
    install_requires=[
        "azure-storage-blob",
-       "iree-base-compiler",
-       "iree-base-runtime",
```

If someone is working with a source build they will have some packages and tools already configured in their environment and having the test package force an install from PyPI may overwrite what they have.